pkg/util/ansible_execution.go

```diff
@@ -119,11 +123,17 @@ func AnsibleExecution(
 			MountPath: "/runner/inventory/hosts",
 			SubPath:   "inventory",
 		}
+		networkConfigMount := corev1.VolumeMount{
+			Name:      "inventory",
+			MountPath: "/runner/network/nic-config-template",
```

Comment:
The constant is defined in the `deployment` package and we aren't importing it here, importing it would create a cyclic import. Since it was only used once here, I just followed what we did for the inventory/hosts file above rather than redefining a non-exported const in the utils package.